Docs/Specifics/Choreo.md

```diff
@@ -0,0 +1,68 @@
+# Choreo
+
+Choreo is a path planning library that uses path-optimization to create time-constrained autonomous paths for the autonomous period. Learning to integrate Choreo into our codebase is important as autonomous, the first 15 seconds of the match, can be the make or break of a high-scoring teleop match.
+
+## Choreo Structure
+
+The following is the stack that is involved to integrate with Choreo:
+
+### Choreo GUI
+
+![Choreo Hero Home Page](/Assets/choreo-hero.png)
+
+The Choreo GUI is the multiplatform Tauri (React + Rust) app that allows you to create and save the paths. It's an independent app from Visual Studio or WPILib (although for the 2025 season it might be bundled in WPILib).
```

Comment:
I would lighten up on the details about how Choreo is built, just cover the essentials like what numerical optimization is and how its applicable to trajopt.